main.py

```diff
@@ -100,17 +145,28 @@ def create_histogram_fig(df, DAYS):
                     [
                         dbc.Col(
                             [
-                                html.H3(
-                                    "Where In NYC Are Cyclists Getting Injured?",
-                                    className="app-header--title my-10",
+                                html.H2(
+                                    "Where In NYC Are Cyclists Getting Hurt?",
                                 ),
                                 html.P(
-                                    "This map displays geospatial data of traffic crash events in NYC in which at least one cyclist was injured, within the past 30 days. Vehicle data and a primary contributing factor are provided where available."
+                                    "This map displays geospatial data of traffic crash events in NYC wherein at least one cyclist was injured, within the past 30 days. Vehicle data and a contributing factor are provided where available."
+                                ),
+                                html.Footer(
+                                    [
+                                        "Data courtesy of the NYC Open Data ",
+                                        html.A(
+                                            "Motor Vehicle Collisions-Crashes",
+                                            href="https://data.cityofnewyork.us/Public-Safety/Motor-Vehicle-Collisions-Crashes/h9gi-nx95/about_data",
+                                            target="_blank",
+                                        ),
+                                        " API, which is stated to receive daily updates. (although it is usually not current up to the present day)",
+                                    ],
                                 ),
                             ],
                             xs=12,
-                            md=2,
-                            lg=2,
+                            sm=12,
```

Comment:
Happy you playing around with breakpoints. This is a deep rabbithole, but happy you got some practice here 👍